Gopkg.toml

```diff
@@ -49,7 +49,7 @@ required = [
 # We need https://github.com/percona/percona-toolkit/pull/396.
 [[constraint]]
   name = "github.com/percona/percona-toolkit"
-  branch = "3.0"
+  branch = "PMM-4278-official-mongo-driver"
```

Comment:
Why there is no PR for that branch?

Reply:
Because we can't merge it. We are depending on https://github.com/percona/percona-toolkit/tree/PMM-4278-official-mongo-driver/src/go/mongolib, and it uses `mgo.v2/bson`. 

As we merged to `mongo-driver` we can't use `mgo.v2/bson` and should use an official one. 

So we talk with @rnovikovP and decide to make the different branch and change `mgo.v2/bson` to `mongo-driver/bson` to make this works.

Reply:
we also should  change  https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/.gitmodules#L42 for  beta3

PT migration will be tracked in a separate issue

Reply:
>> Why there is no PR for that branch?

> Because we can't merge it.

But it doesn't mean we can't create it. Tracking PRs is easier.